### DISCUSSION TITLE
Off-topic

### DISCUSSION THREAD

**Discussion by mapleexists** [FrozenBlock/WilderWild#144] [General]
stuff that's not really related to wilder wild goes here

**Comment by Osmiooo** [comment]
first

**Comment by mapleexists** [comment]
have you guys ever broken the heel off of a high heel

**Comment by Osmiooo** [comment]
yes

**Reply by mapleexists** [reply]
> yes

have you really

**Reply by mapleexists** [reply]
i meant like, while wearing it

**Reply by mapleexists** [reply]
it fucking hurts btw

**Reply by Osmiooo** [reply]
> > yes
> 
> have you really

idk i didnt read

**Reply by mapleexists** [reply]
marble

**Reply by Osmiooo** [reply]
![wilder wild](https://user-images.githubusercontent.com/75801667/189430969-0c57cfb3-06e2-49b0-a1f5-e5cebb250c76.png)

**Reply by mapleexists** [reply]
> ![wilder wild](https://user-images.githubusercontent.com/75801667/189430969-0c57cfb3-06e2-49b0-a1f5-e5cebb250c76.png)

walter wight

**Comment by Osmiooo** [comment]
![1fab1](https://user-images.githubusercontent.com/75801667/189431222-8d9a9172-60d2-4348-ba3d-59c6cfca8d52.png)

**Comment by Osmiooo** [comment]
gonna die rq brb

**Reply by AViewFromTheTop** [reply]
follow instructions carefully
microwave for one minute on "Slags" setting

**Reply by Osmiooo** [reply]
i dont have a mcirowcave

**Reply by mapleexists** [reply]
> i dont have a mcirowcave

wait is that a joke

**Reply by Osmiooo** [reply]
> > i dont have a mcirowcave
> 
> wait is that a joke

wdym

**Reply by AViewFromTheTop** [reply]
no

**Reply by Osmiooo** [reply]
> > i dont have a mcirowcave
> 
> wait is that a joke

if you're talking about the spelling mistake with cave at the end, yes that's a joke (cave wasn't meant)
if you're talking about going to die real quick, then idk you chose

**Reply by mapleexists** [reply]
im talking about you not having a microwave you completely missed the intent of my question lmao

**Reply by mapleexists** [reply]
> no

didnt know you spoke spanish
![image](https://user-images.githubusercontent.com/27739930/189432900-24c7de5d-880b-4449-bee3-d3db0b4b0ddc.png)

**Reply by Osmiooo** [reply]
oh well
yes that's true i dont have one

**Reply by Osmiooo** [reply]
> > no
> 
> didnt know you spoke spanish ![image](https://user-images.githubusercontent.com/27739930/189432900-24c7de5d-880b-4449-bee3-d3db0b4b0ddc.png)

LOL

**Reply by mapleexists** [reply]
> oh well yes that's true i dont have one

wtf how u make pizza rolls 😭

**Reply by AViewFromTheTop** [reply]
pizza rolls fucking SUCK

**Reply by mapleexists** [reply]
> pizza rolls fucking SUCK

i know but i couldnt think of anything else that you can make in a microwave LMAO
frozen pizza in general is disgusting

**Reply by AViewFromTheTop** [reply]
jimmy dean delights

**Reply by AViewFromTheTop** [reply]
hamsters and crows

**Reply by AViewFromTheTop** [reply]
infants

**Reply by AViewFromTheTop** [reply]
[REDACTED]

**Reply by AViewFromTheTop** [reply]
cheese

**Reply by AViewFromTheTop** [reply]
soap

**Reply by AViewFromTheTop** [reply]
Speed of Sound and Clocks

**Reply by mapleexists** [reply]
> jimmy dean delights

oh thats true
jimmy dean breakfast stuff hits harder than it should tbh

**Reply by AViewFromTheTop** [reply]
so does A View From The Top

**Reply by AViewFromTheTop** [reply]
and Clocks

**Reply by mapleexists** [reply]
haha osmio how does it feel to know that you can't make plasma by putting two grapes (or other equally sized round objects whose diameters are similar to the wavelength of the microwave) next to one another in the microwave

**Reply by AViewFromTheTop** [reply]
but only that specific jimmy dean thing
i don't like how it tastes but it fills my apparently non existent stomach for the entire day

**Reply by AViewFromTheTop** [reply]
tbf a chocolate chip is literally a fucking meal for me so

**Reply by mapleexists** [reply]
yeah there are some jimmy dean things that are kinda gross
but otherwise mmmmmmm

**Reply by mapleexists** [reply]
> tbf a chocolate chip is literally a fucking meal for me so

tf do you have hyperglycaemia

**Reply by AViewFromTheTop** [reply]
what's that

**Reply by mapleexists** [reply]
it's where your resting blood sugar levels are higher than they should be
i have the opposite of it which is where your resting blood sugar levels are lower than they should be, which means i can pass out if i don't eat enough in a single day
i am assuming the opposite of it would be that you don't need to eat as much stuff per day, else you'd risk giving yourself diabetes

**Reply by Osmiooo** [reply]
> > oh well yes that's true i dont have one
> 
> wtf how u make pizza rolls 😭

i dont

**Reply by AViewFromTheTop** [reply]
idk i just never feel hungry unless i haven't eaten in over a day
so i just have to set up two specific times and just eat so i dont die

**Reply by mapleexists** [reply]
> i dont

based

but seriously like how do you not have a microwave
don't say it's because you live in france i know that's a lie i lived in france for like two years and everyone i knew had a microwave

**Reply by AViewFromTheTop** [reply]
some days i DO actually feel hungry but it goes away so quickly i forget about it
it's weird

**Reply by mapleexists** [reply]
> some days i DO actually feel hungry but it goes away so quickly i forget about it it's weird

bro what
do you have an inverse tapeworm that just feeds you whenever you're hungry

**Reply by AViewFromTheTop** [reply]
he's probably not allowed to use it

**Reply by Osmiooo** [reply]
> but seriously like how do you not have a microwave don't say it's because you live in france i know that's a lie i lived in france for like two years and everyone i knew had a microwave

uh i have an oven and thats better imo
my parents hate microwaves, especially my mom
so we never had one and since i grew up with that then i wont have one either
thats just my familly tbh, not a common thing in france as u said

**Reply by AViewFromTheTop** [reply]
merp how tf am i supposed to know

**Reply by AViewFromTheTop** [reply]
why tf do they hate microwaves

**Reply by Osmiooo** [reply]
idk it's just useless since we have an oven

**Reply by mapleexists** [reply]
> > but seriously like how do you not have a microwave don't say it's because you live in france i know that's a lie i lived in france for like two years and everyone i knew had a microwave
> 
> uh i have an oven and thats better imo my parents hate microwaves, especially my mom so we never had one and since i grew up with that then i wont have one either thats just my familly tbh, not a common thing in france as u said

oh ok
and yeah ovens are generally better and more versatile
you can toast bread in an oven
you can cook a pizza in an oven
~~you can cancel a child in an oven~~ you can bake a potato in an oven
microwaves make stuff soggy because they use radiation to cook stuff while ovens just use heat

**Reply by mapleexists** [reply]
> merp how tf am i supposed to know

idk but you probably should get that looked at
kinda concerning

**Reply by Osmiooo** [reply]
> ~you can cancel a child in an oven~ 

sadly its harder to kill someone with an oven

**Reply by AViewFromTheTop** [reply]
i'd use a leather boot for that not an oven

**Reply by AViewFromTheTop** [reply]
it's not worth looking at this has been going on for a few years and i haven't died
i'm just used to it now

**Reply by mapleexists** [reply]
> > ~you can cancel a child in an oven~
> 
> sadly its harder to kill someone with an oven

it's not though
the most a microwave can do to a person is give them cancer because microwaves are only effective against small objects lol
and while i guess that does kill a person it's far slower than it would be to kill a person in an oven
jesus christ what is this conversation

**Reply by Osmiooo** [reply]
guys do u have some advice to make the kids in my basement shut up?
they won't stop yelling and screaming all day long, it's very annoying

**Reply by mapleexists** [reply]
> it's not worth looking at this has been going on for a few years and i haven't died i'm just used to it now

i guess so

**Reply by AViewFromTheTop** [reply]
put on spongebob

**Reply by Osmiooo** [reply]
> it's not though the most a microwave can do to a person is give them cancer because microwaves are only effective against small objects lol and while i guess that does kill a person it's far slower than it would be to kill a person in an oven jesus christ what is this conversation

kinda sus
but ok i'll try

**Reply by mapleexists** [reply]
> guys do u have some advice to make the kids in my basement shut up? they won't stop yelling and screaming all day long, it's very annoying

pacific ocean

**Reply by Osmiooo** [reply]
> > guys do u have some advice to make the kids in my basement shut up? they won't stop yelling and screaming all day long, it's very annoying
> 
> pacific ocean

best advice thanks

**Reply by mapleexists** [reply]
oh uh yeah put on spongebob

**Reply by AViewFromTheTop** [reply]
hey that's where i put you
btw how did you survive that

**Reply by mapleexists** [reply]
ok im gonna try to find a way to make it so that discussions update live this is getting ridiculous

**Reply by Osmiooo** [reply]
> hey that's where i put you btw how did you survive that

can you put me in next time? would be easier to die with that kind of help

**Reply by mapleexists** [reply]
> hey that's where i put you btw how did you survive that

who

**Reply by Osmiooo** [reply]
> oh uh yeah put on spongebob

tv makes noises that would still annoy me
less but still

**Reply by AViewFromTheTop** [reply]
oh how can you be annoyed by s1-s3 SpongeBob's voice

**Reply by AViewFromTheTop** [reply]
and the episodes of season 4 that were produced in 2004 too

**Reply by Osmiooo** [reply]
im annoyed by sounds
nothing in particular
especially if it comes from my basement, it would be very tiny sound but loud enough to be heard and that would be annoying as fuck

**Reply by AViewFromTheTop** [reply]
warden lookin ass

**Reply by Osmiooo** [reply]
maybe i can kill one of them so that the other one will learn that they need to shut up

**Reply by AViewFromTheTop** [reply]
in minecraft

**Reply by Osmiooo** [reply]
what? no

**Reply by AViewFromTheTop** [reply]
in pvzgw

**Reply by Osmiooo** [reply]
irl

**Reply by AViewFromTheTop** [reply]
well yeah pvzgw does exist irl it's a game

**Reply by Osmiooo** [reply]
do i exist irl

**Reply by AViewFromTheTop** [reply]
yes

**Reply by Osmiooo** [reply]
fuck
how do i unexist

**Reply by AViewFromTheTop** [reply]
watch spongebob

**Reply by Osmiooo** [reply]
spongebob makes me unexist?

**Reply by AViewFromTheTop** [reply]
cel

**Reply by Osmiooo** [reply]
no

**Reply by AViewFromTheTop** [reply]
paint

**Comment by AViewFromTheTop** [comment]
sorry but i don't wear high heels
i either wear black shoes that i don't care about, or black leather boots i can crush the skulls of my enemies with

**Reply by mapleexists** [reply]
good, dont wear high heels (particularly stilettos)
there is absolutely nothing good about them other than the fact that they make the wearer taller and they look kind of pretty
the amount of times i've nearly twisted my ankles and the amount of times i've sprained my ankles by wearing stilettos is astronomical

**Reply by AViewFromTheTop** [reply]
why tf would i wear them i know they're shit
i don't understand why anybody wears them

**Reply by mapleexists** [reply]
i dont either i hate them
i feel like stilettos were invented for the sole purpose of benefitting men by giving them something to look at rather than benefitting women by making them actually comfortable
i mean obviously women arent the only ones who can wear them but they were invented for women
if you ever suddenly decide that you're trans please never let yourself wear them

**Reply by AViewFromTheTop** [reply]
sounds like a lot of things made, caused, and done by the average american conservative male

**Reply by mapleexists** [reply]
then again, im kinda quirky and tend to break gender norms a lil bit so maybe it's just me, idk
point is that i fucking hate stilettos

**Reply by mapleexists** [reply]
> sounds like a lot of things made, caused, and done by the average american conservative male

BASED

**Reply by AViewFromTheTop** [reply]
i'm not sure if i should send my other response

**Reply by mapleexists** [reply]
you should

**Reply by Osmiooo** [reply]
you should

**Reply by AViewFromTheTop** [reply]
idk man

**Reply by Osmiooo** [reply]
do it

**Reply by AViewFromTheTop** [reply]
no

**Reply by mapleexists** [reply]
tell me privately and i'll gauge whether or not it's okay to post here

**Comment by mapleexists** [comment]
this is fun btw it feels like we're actually chatting like we would in an actual chatroom because there's no longer any stupid restrictions to the things we can say

**Reply by AViewFromTheTop** [reply]
there's a lot of restriction on me wdym

**Reply by AViewFromTheTop** [reply]
also discussions don't update live unlike commit comments and issue comments so you have to keep reloading

**Reply by mapleexists** [reply]
> also discussions don't update live unlike commit comments and issue comments so you have to keep reloading

those have never updated live for me

**Reply by mapleexists** [reply]
> there's a lot of restriction on me wdym

i mean on discord we have channels that only we can access ;)
im kidding ive been done trying to get you to join discord for months

**Reply by AViewFromTheTop** [reply]
please don't mention that shit website especially not now

**Reply by mapleexists** [reply]
sorry sorry i didnt mean it i was joking

**Reply by mapleexists** [reply]
i think there might be a way that we can create a discussion that only we can view though

**Reply by Osmiooo** [reply]
holy shit

**Reply by AViewFromTheTop** [reply]
me realizing i could draw spongebob and escape reality for the first time

**Reply by AViewFromTheTop** [reply]
me realizing squidward is painful as all hell to draw

**Reply by AViewFromTheTop** [reply]
me realizing sandy is impossible to draw

**Reply by AViewFromTheTop** [reply]
me realizing mrs. puff's proportions are impossible to get down correctly

**Reply by mapleexists** [reply]
> holy shit

?

**Reply by Osmiooo** [reply]
> > holy shit
> 
> ?

im still not dead

**Reply by Osmiooo** [reply]
weird

**Reply by AViewFromTheTop** [reply]
try the microwave
OH WAIT

**Reply by Osmiooo** [reply]
you know you can buy braincells

**Reply by AViewFromTheTop** [reply]
braincells at a cheap price!
satisfactory

**Comment by EthanRStokes** [comment]
Among Xfrtrex

**Reply by AViewFromTheTop** [reply]
ew

**Reply by Osmiooo** [reply]
ew

**Reply by AViewFromTheTop** [reply]
GUYS THIS IS COMPLETELY UNRELATED TO EVERYTHING BUT I JUST HAD A JONNY BUCKLAND MOMENT

**Reply by AViewFromTheTop** [reply]
WHEN LIFE IS BEING A BITCH TO ME I MAKE IT WORTH LIVING FOR MYSELF I LOVE THIS
I AM A GUITARIST

**Reply by AViewFromTheTop** [reply]
meh not so good anymore

**Reply by AViewFromTheTop** [reply]
Enjoy the unknown
Always, always
Enjoy the unknown
Always, always
The end is near, We’re Dying Here
Always, always
Enjoy the unknown

**Reply by Osmiooo** [reply]
what

**Reply by AViewFromTheTop** [reply]
Your smile on TV means nothing to me,
Cause you are all you could be
And I feel sorry about the death,
Of your pet

Enjoy the unknown
Always, always
Enjoy the unknown
Always, always
The end is near, we’re dying here
Always, always
Enjoy the unknown
Creep around your window

**Reply by Osmiooo** [reply]
totally agree

**Reply by AViewFromTheTop** [reply]
I think freezing water just went straight into my lung

**Reply by Osmiooo** [reply]
skill issue

**Reply by AViewFromTheTop** [reply]
Ok you know what screw it I’m gonna ask publicly I’ve been waiting too long to ask
Have I been too overbearing on this mod? Have I been unreasonable, stupid, irritating, and unjustified? Be honest.

**Reply by mapleexists** [reply]
how can you be overbearing when you're the sole reason why the mod is as good as it is

**Reply by AViewFromTheTop** [reply]
> how can you be overbearing when you're the sole reason why the mod is as good as it is

There are so many places I could being going wrong and so many people I could be upsetting
This isn’t even my mod to begin with, and I feel like everyone hates me when I’m actively working on the mod
I’m being a dick to everyone and it makes me feel terrible

**Comment by mapleexists** [comment]
@AViewFromTheTop random request, could you post Back to your youtube channel and whatever other thingies you post your music to? i'm trying to compile a youtube playlist of the mod's soundtrack since that doesn't already exist for whatever reason

**Reply by mapleexists** [reply]
actually nvm i'll just post it to the frozenblock channel and add you to the description

**Reply by EthanRStokes** [reply]
> actually nvm i'll just post it to the frozenblock channel and add you to the description

hold up theres a frozenblock channel?

**Reply by mapleexists** [reply]
yeah but it doesnt have anything on it yet

**Reply by AViewFromTheTop** [reply]
Sorry but I can’t just add official music to YT, I have to pay about $100 for that and it gets royalties
That’s a sure fire way to piss people off, isn’t it?

**Reply by AViewFromTheTop** [reply]
Oh also I just woke up and I don’t think I can think straight

**Reply by AViewFromTheTop** [reply]
Also could you cut Death Ambience out of The Other Side

**Reply by mapleexists** [reply]
wha?

**Reply by mapleexists** [reply]
wdym by death ambience

**Reply by mapleexists** [reply]
also, i've already uploaded it https://youtu.be/PPSQ5UCZzKE

**Reply by AViewFromTheTop** [reply]
NO NOT DEATH AMBIENCE ITS NOT SUPPOSED TO BE ON YOUTUBE YET

**Reply by mapleexists** [reply]
if you're talking about the chord at the beginning, it honestly doesn't matter since it's only a very short section of it

**Reply by AViewFromTheTop** [reply]
YES I AM AND IT MEANS A LOT

**Reply by mapleexists** [reply]
aaaaaaa ok fine i'll remove it and reupload the video

**Comment by mapleexists** [comment]
mario

**Reply by mapleexists** [reply]
![image](https://user-images.githubusercontent.com/27739930/191792274-c1a7a4ab-3f5d-47d8-8db5-baf03c9b368f.png)

**Reply by mapleexists** [reply]
🍔 <- mario
